Gruntfile.js

```diff
@@ -191,7 +191,7 @@ module.exports = function(grunt) {
   grunt.registerTask("release:minor", ["bump:minor", "dist-compile", "gitcommit:version"]);
   grunt.registerTask("release:major", ["bump:major", "dist-compile", "gitcommit:version"]);
 
-  grunt.registerTask("dist-compile", ["test", "uglify", "compress"]);
+  grunt.registerTask("dist-compile", ["test", "exec:npm:sed-version", "uglify", "compress"]);
```

Comment:
this is awkward, we have an NPM task which calls a grunt task which then calls an NPM task? can you think of a better way to organize this...